package.json

```diff
@@ -76,77 +76,82 @@
     "@testing-library/react-hooks": "^3.4.2",
     "@types/classnames": "^2.2.11",
     "@types/core-js": "^2.5.4",
     "@types/js-cookie": "^2.2.6",
-    "@types/lodash": "^4.14.160",
-    "@types/react": "^16.9.56",
+    "@types/lodash": "^4.14.165",
+    "@types/react": "^16.14.2",
     "@types/react-document-title": "^2.0.4",
-    "@types/react-dom": "^16.9.9",
+    "@types/react-dom": "^16.9.10",
     "@types/react-redux": "^7.1.11",
     "@types/react-router-dom": "^5.1.6",
     "@types/redux-logger": "^3.0.8",
     "@types/uuid": "^8.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.7.0",
-    "@typescript-eslint/parser": "^4.7.0",
-    "autoprefixer": "^9.8.5",
+    "@typescript-eslint/eslint-plugin": "^4.9.0",
+    "@typescript-eslint/parser": "^4.9.0",
+    "autoprefixer": "^10.0.4",
     "babel-jest": "^26.6.3",
-    "babel-loader": "^8.2.1",
+    "babel-loader": "^8.2.2",
     "clean-webpack-plugin": "^3.0.0",
-    "core-js": "^3.7.0",
+    "core-js": "^3.8.0",
     "coveralls": "^3.1.0",
-    "css-loader": "^3",
-    "eslint": "^7.13.0",
+    "css-loader": "^5.0.1",
+    "eslint": "^7.14.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-import-resolver-typescript": "^2.3.0",
     "eslint-import-resolver-webpack": "^0.13.0",
     "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.1.0",
+    "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-simple-import-sort": "^5.0.3",
-    "fetch-mock": "^9.10.4",
+    "eslint-plugin-simple-import-sort": "^6.0.1",
+    "fetch-mock": "^9.11.0",
     "file-loader": "^6.2.0",
     "html-webpack-plugin": "^4.5.0",
     "husky": "^4.3.0",
     "i18next-scanner-webpack": "^0.6.0",
+    "isomorphic-fetch": "^3.0.0",
     "jest": "^26.6.3",
     "jest-junit": "^12.0.0",
-    "mini-css-extract-plugin": "^1.3.0",
+    "mini-css-extract-plugin": "^1.3.1",
     "node-fetch": "^2.6.1",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
-    "postcss-loader": "^3.0.0",
-    "prettier": "^2.1.2",
-    "react-test-renderer": "^16.13.1",
+    "postcss": "^8.1.10",
+    "postcss-loader": "^4.1.0",
+    "prettier": "^2.2.1",
+    "react-test-renderer": "^16.14.0",
     "redux-mock-store": "^1.5.4",
     "sass": "^1.29.0",
     "sass-loader": "^10.1.0",
-    "stylelint": "^13.7.2",
+    "stylelint": "^13.8.0",
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-recommended": "^3.0.0",
     "stylelint-prettier": "^1.1.2",
     "stylelint-scss": "^3.18.0",
     "svg-inline-loader": "^0.8.2",
-    "typescript": "^4.0.5",
+    "typescript": "^4.1.2",
     "url-loader": "^4.1.1",
     "webpack": "^4.44.2",
     "webpack-cli": "^4.2.0",
     "webpack-dev-server": "^3.11.0",
-    "webpack-merge": "^5.3.0"
+    "webpack-merge": "^5.4.0"
+  },
+  "resolutions": {
+    "@types/react": "^16.14.2"
```

Comment:
Because we're not upgrading to react v17 yet, this prevents duplicate type definitions for react. For the record, here are the js deps we're intentionally not upgrading yet:

```
 devDependencies
   name                 range   from        to      url
 ◯ @types/react         latest  16.14.2  ❯  17.0.0  https://github.com/DefinitelyTyped/DefinitelyTyped.git
 ◯ @types/react-dom     latest  16.9.10  ❯  17.0.0  https://github.com/DefinitelyTyped/DefinitelyTyped.git
 ◯ react-test-renderer  latest  16.14.0  ❯  17.0.1  https://reactjs.org/
 ◯ webpack              latest  4.44.2   ❯  5.9.0   https://github.com/webpack/webpack

 dependencies
   name                 range   from        to      url
 ◯ react                latest  16.14.0  ❯  17.0.1  https://reactjs.org/
 ◯ react-dom            latest  16.14.0  ❯  17.0.1  https://reactjs.org/
```